### ISSUE TITLE
DFG frequency visualizer throws `NameError: name 'stat_locale' is not defined`

### ISSUE THREAD

**Issue by lmoustak** [process-intelligence-solutions/pm4py#285]
Running this (code shortened for brevity):
```python
from pm4py.visualization.dfg import visualizer as dfg_visualizer
# ...
gviz = dfg_visualizer.apply(dfg, log=log, variant=dfg_visualizer.Variants.FREQUENCY, parameters=parameters)
```
will eventually print this Traceback:
```
Traceback (most recent call last):
  File "d:/<user>/Documents/UoA/DataScienceAndInformationTechnologies/Thesis/ProcessMining/Sessions&Clustering/2/mineSubset2.py", line 150, in <module>
    gviz = dfg_visualizer.apply(
  File "C:\Users\<user>\anaconda3\lib\site-packages\pm4py\visualization\dfg\visualizer.py", line 66, in apply
    return exec_utils.get_variant(variant).apply(dfg, log=log, activities_count=activities_count, soj_time=soj_time, parameters=parameters)
  File "C:\Users\<user>\anaconda3\lib\site-packages\pm4py\visualization\dfg\variants\frequency.py", line 336, in apply
    return graphviz_visualization(activities_count, dfg, image_format=image_format, measure="frequency",
  File "C:\Users\<user>\anaconda3\lib\site-packages\pm4py\visualization\dfg\variants\frequency.py", line 223, in graphviz_visualization
    stat_string = human_readable_stat(soj_time[act], stat_locale)
NameError: name 'stat_locale' is not defined
```

`stat_locale` in `graphviz_visualization()` is not initialized. This issue does not apply to the Performance variant, as the `stat_locale` keyword parameter is defined. Nor does it apply while calling `pm4py.save_vis_dfg()` despite using the Frequency variant.

Changing the Frequency variant functions to include this parameter (defaulting to `{}`) like in the Performance variant fixes this issue.

**Comment by fit-alessandro-berti** [comment]
Dear lmoustak, thanks for signaling.

We will release a fix for this in the next release.

Have a nice day.

**Comment by fit-alessandro-berti** [comment]
Dear lmoustak, the issue has been resolved in release 2.2.16
